src/components/shared/navigation-bar.tsx

```diff
@@ -25,24 +26,39 @@ const ButtonLinkOutlined = styled.div`
   color: var(--primary);
 `;
 
-export const NavigationBar: React.FunctionComponent = () => (
-  <nav className="container mx-auto pt-4 px-4">
-    <div className="flex flex-wrap items-center">
-      <div className="w-auto mr-auto">
-        <Logo>
-          <Link to="/">verify.gov.sg</Link>
-        </Logo>
-      </div>
-      <div className="w-auto">
-        <div className="flex items-center">
-          <Link to="/faq">
-            <FaqLink className="mr-8">FAQ</FaqLink>
-          </Link>
-          <Link to="/verify">
-            <ButtonLinkOutlined>Verify</ButtonLinkOutlined>
-          </Link>
+interface NavigationBarProps {
+  setVerificationStatus?: (number: Status) => void;
```

Comment:
suggestion => let's inverse the dependency here:
- to rename `setVerificationStatus` to `onVerifyLinkClicked`
- to accept NO parameters
- to call `setVerificationStatus(Status.IDLE)` from verify-page when `onVerifyLinkClicked` is called